frontend/src/config.js

```diff
@@ -1 +1,19 @@
+import axios from "axios";
+
 export const BASE_URL = "http://localhost:8080/api";
+
+export const ApiRequest = () => {
+  return axios.create({
+    baseURL: BASE_URL,
+    headers: {
+      "Content-type": "application/json",
+    },
+  });
+};
+
+export const ApiTokenRequest = (token) => {
+  return axios.create({
+    baseURL: BASE_URL,
+    headers: { Authorization: `Bearer ${token}` },
+  });
+};
```

Comment:
Could you merge this (in case you always need application/json content type, Authorization header could be set conditionally)?